client/my-sites/upgrades/checkout/free-trial-confirmation-box.js

```diff
@@ -40,8 +42,14 @@ var FreeTrialConfirmationBox = React.createClass( {
 		);
 	},
 
-	render: function() {
-		var classSet = classNames( {
+	getProductName() {
+		const planProduct = find( this.props.cart.products, isPlan );
+
+		return ( planProduct && planProduct.product_name ) || '';
+	},
+
+	render() {
+		const classSet = classNames( {
 			'credits-payment-box': true,
 			selected: this.props.selected === true
 		} );
```

Comment:
Personally I would render null when `this.props.selected` is set to `false`, but I didn't want to touch it in this PR.